server/models/Request.js

```diff
@@ -4,11 +4,11 @@ const requestSchema = new mongoose.Schema({
   userId: {
     type: mongoose.Schema.Types.ObjectId,
     required: true,
-    ref: "User",
+    ref: "profiles",
```

Comment:
Please ensure that this `ref` matches the model name for `profile`. Is it pluralized?

Reply:
Can we address this comment here - should be "Profile"